myplanet/skeletor
# SKEL-8: Add linting to the project

How to test:

Just enter an invalid CSS rule/property value into a SCSS file and run

```yarn dev```

It should warn what's the problem.
